templates/organization/package/details.html.twig

```diff
@@ -20,6 +20,13 @@
             <div class="col-6">
                 <h4>Package name</h4>
                 <p>{{ package.name }}</p>
+                {% if package.url %}
+                    <p>
+                        <small class="font-italic" style="font-size: 0.7em">
+                            <a href="{{ package.url }}" target="_blank" rel="nofollow noopener noreferrer">{{ package.url }}</a>
+                        </small>
+                    </p>
+                {% endif %}
```

Comment:
What about something like this:
![Screenshot from 2021-06-11 09-00-24](https://user-images.githubusercontent.com/8239917/121644596-8082ef80-ca93-11eb-98bf-d062f2e829e7.png)

Reply:
```html
        <div class="row">
            <div class="col-3">
                <h4>Package name</h4>
                <p>{{ package.name }}</p>
            </div>
            <div class="col-3">
                <h4>Repository URL</h4>
                <p><a href="{{ package.url }}" target="_blank" rel="nofollow noopener noreferrer">{{ package.url }}</a></p>
            </div>
            <div class="col-6">
                <h4>Description</h4>
                <p>{{ package.description }}</p>
            </div>
        </div>
```

Reply:
lets do this :)